### ISSUE TITLE
Question about training without pretrained weight

### ISSUE THREAD

**Issue by HiHippie** [Han-Jia/UNICORN-MAML#1]
I'm a huge fan of your excellent work!

Have you tried to train UNICORN MAML without using pre-training weights? I found that if I do not use the pre-training weights, the training result is only 26%+. Thus,  could you please provide the parameter settings in this case ?

Many thanks~

**Comment by Han-Jia** [comment]
Hello,

In our ICLR paper, we only investigated MAML based on the pre-trained weights, which is necessary to reproduce most SOTA few-shot learning methods.

Based on our experience, when meta-training without the pre-trained weights, we need to use a larger learning rate and much more meta-training epochs.

**Comment by jihoontack** [comment]
Hi, thank you for the awesome work! Got inspired a lot!

I also have a question about the pre-trained weights.

What kind of data augmentation techniques were used for the pre-training step? 
There are two augmentation details in your code, and it seems like the one without color augmentation was used. (I just wanted to make it clear) 

Thank you very much for your time!
Best,
Jihoon

**Comment by Han-Jia** [comment]
Hi, Jihoon,

We directly use the pre-trained weights from FEAT. The pre-training configurations are [listed here](https://github.com/Sha-Lab/FEAT/blob/47bdc7c1672e00b027c67469d0291e7502918950/model/dataloader/mini_imagenet.py#L50). 

Colorjittering is used, but based on my experience, it will not influence the performance too much. Only using the RandomResizedCrop also gets good results.
